backend/compact-connect/lambdas/python/provider-data-v1/tests/function/test_handlers/test_providers.py

```diff
@@ -431,3 +468,73 @@ def test_get_provider_ssn_forbidden_without_correct_jurisdiction_level_scope(sel
         resp = get_provider_ssn(event, self.mock_context)
 
         self.assertEqual(403, resp['statusCode'])
+
+    def test_get_provider_ssn_throttled_and_deactivated_if_staff_user_goes_beyond_rate_limit(self):
+        """
+        The staff user has called this endpoint more than the set limit, so the endpoint returns a 429
+        If the user makes another request after this, their account should be deactivated and an alert should be
+        fired.
```

Comment:
Small suggestion to wording here- from reading, my assumption was that the last request was not actually part of this test.

```suggestion
        The staff user has called this endpoint more than the set limit, so the endpoint returns a 429
        and, after one more request, their account is deactivated and an alert is
        fired.
```